svc-cloud-transport/pom.xml

```diff
@@ -289,15 +290,16 @@
                     <appServicePlanName>${functionServicePlan}</appServicePlanName>
                     <subscriptionId>${subscriptionID}</subscriptionId>
                     <region>${functionRegion}</region>
-
+		    
```

Comment:
need to add the <auth> tag like the other functions https://github.com/microsoft/azure-maven-plugins/wiki/Authentication#service-principles-in-settings.xml